src/libraries/Microsoft.Extensions.Configuration.EnvironmentVariables/src/Microsoft.Extensions.Configuration.EnvironmentVariables.csproj

```diff
@@ -2,7 +2,10 @@
 
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
+    <IsPackable>true</IsPackable>
     <EnableDefaultItems>true</EnableDefaultItems>
+    <NoWarn>$(NoWarn);NU5128</NoWarn>
```

Comment:
Why is this necessary? I would not expect to have a file present for a tfm without a dependency node.

Reply:
we can remove this now, earlier it was requires when i was not removing the dependency node

Reply:
we can remove this now, earlier it was requires when i was not removing the dependency node

Reply:
Let's please agree on that we never accept this NoWarn to be set as it indicates a wrong use of the Pack task.